rasa_core/server.py

```diff
@@ -9,14 +9,14 @@
 from flask_cors import CORS, cross_origin
 from flask_jwt_simple import JWTManager, view_decorators
 
+import rasa
```

Comment:
is this also in python sys path if you build & run the docker image?

Reply:
yes, if you run `pip install rasa` (or `pip install -e .`) it will install two packages, `rasa` and `rasa_core`